skimage/filters/thresholding.py

```diff
@@ -1177,3 +1178,69 @@ def threshold_multiotsu(image, classes=3, nbins=256):
     thresh = bin_centers[thresh_idx]
 
     return thresh
+
+
+def threshold_phansalkar(image, window_size=15, k=0.25, r=None, p=2.0, q=10.0):
+    """Applies Phansalkar local threshold to an array. Phansalkar is a
+    modification of Sauvola technique to deal with low contrast images.
```

Comment:
@kleinerELM not yet fixed — the short description should fit in one line, followed by a blank line, followed by a longer explanation. In this case:

```suggestion
    """Find the Phansalkar local threshold for an array.
    
    Phansalkar is a modification of Sauvola thresholding designed to deal with
    low contrast images.
```